services/resource-coordinator/src/drivers/kubernetes/KubernetesRunningNode.js

```diff
@@ -0,0 +1,52 @@
+const _ = require('lodash');
+const { RunningNode } = require('@openintegrationhub/resource-coordinator');
+
+class KubernetesRunningNode extends RunningNode {
```

Comment:
Same as with k8s driver. Maybe this is generic enough to be placed in src/resource-coordinator?